src/scenarios/shared/util.py

```diff
@@ -35,4 +36,9 @@ def uploadtokenpresent():
     return environ.get(UPLOAD_TOKEN_VAR) is not None
 
 def runninginlab():
-    return environ.get('PERFLAB_INLAB') is not None
\ No newline at end of file
+    return environ.get('PERFLAB_INLAB') is not None
+
+def getruntimeidentifier():
+    rid = 'win-' if sys.platform == 'win32' else 'linux-'
+    rid += 'x64' if platform.machine().endswith('64') else 'x86'
```

Comment:
This will return `linux-x64` on our ARM64 machines.

Reply:
Updated.